src/arrays/ArrayForSorting.groovy

```diff
@@ -0,0 +1,24 @@
+package arrays
+
+class ArrayForSorting {
+
+     static SortingArray sortingArrayMethod(int[] array) {
+        /*Внешний цикл каждый раз сокращает фрагмент массива,
+          так как внутренний цикл каждый раз ставит в конец
+          фрагмента максимальный элемент*/
+        for (int i = array.length - 1; i > 0; i--) {
+            for (int j = 0; j < i; j++) {
+                /*Сравниваем элементы попарно,
+                  если они имеют неправильный порядок,
+                  то меняем местами*/
+                if (array[j] > array[j + 1]) {
+                    int tmp = array[j];
+                    array[j] = array[j + 1];
+                    array[j + 1] = tmp;
+                }
+            }
+
+        }
+        return new SortingArray(array)
```

Comment:
Is it compiled? SortingArray constructor has two parameters, why do we pass only one?